.github/workflows/codeql.yml

```
@@ -103,7 +103,23 @@ jobs:
     name: PR Semver Labels
     if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      issues: write
     steps:
+      # Checkout is required for local composite actions
+      - uses: actions/checkout@v4
+        if: github.actor == 'dependabot[bot]'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)